src/pages/admin/UsersAdmin.jsx

```diff
@@ -0,0 +1,76 @@
+import { useRequest } from "ahooks";
+import { useEffect, useMemo } from "react";
+import { toast } from "react-hot-toast";
+import { NavLink, useParams } from "react-router-dom";
+
+import Spin from "@/components/Spin";
+import UserView from "@/components/user/UserView";
+import { ROLES } from "@/constants";
+import { useAdmin } from "@/models";
+
+const menu = [
+  { url: ROLES.CLIENT, name: "Clients" },
+  { url: ROLES.EMPLOYER, name: "Employers" },
+  { url: ROLES.ADMIN, name: "Administrators" },
+];
+
+export default function JobsAdmin() {
+  const { tabUrl } = useParams();
+  const { listUsers } = useAdmin();
+  const { data, run, loading } = useRequest(async () => listUsers(), {
```

Comment:
if you want to do the refresh, there would one more proper one `refresh`, useRequest has that.

Reply:
ah thanks, changed